Sources/AblyChat/RoomOptions.swift

```diff
@@ -1,9 +1,31 @@
 import Foundation
 
+/**
+ * Represents the options for a given chat room.
+ */
 public struct RoomOptions: Sendable, Equatable {
+    /**
+     * The presence options for the room. To enable presence in the room, set this property. You may
```

Comment:
should be "... Alternatively, you may use ``RoomOptions/allFeaturesEnabled``..." imo

Reply:
updated